File: Java/Hardware.java
// Hardware merupakan turunan Product
class Hardware extends Product {
    // atrribute Hardware
    private String brand;
    private String model;

    // contructor
    public Hardware() {
    }

    public Hardware(String brand, String model) {
        this.brand = brand;
        this.model = model;
    }

    // getter setter brand
    public String getBrand() {
        return brand;
    }

    public void setBrand(String brand) {
        this.brand = brand;
    }

    // getter setter model
    public String getModel() {
        return model;
    }

    public void setModel(String model) {
        this.model = model;
    }

    // method untuk emnampilkan semua atrribute Hardware
    public void printHardware() {
        System.out.println("Brand : " + getBrand());
        System.out.println("Model : " + getModel());
    }
}

File: python/Hardware.py
from Product import Product
# Hardware merupakan turunan dari Product
class Hardware(Product):
    # contructor
    def __init__(self):
        self.brand = ""
        self.model = ""
    # getter setter brand
    def getBrand(self):
        return self.brand
    
    def setBrand(self, brand):
        self.brand = brand
    # getter setter model
    def getModel(self):
        return self.model

    def setModel(self, model):
        self.model = model
    # method untuk menampilkan semua attribute Hardware
    def printHardware(self):
        print("Brand: " + self.getBrand())
        print("Model: " + self.getModel())